### DISCUSSION TITLE
Highlighting text

### DISCUSSION THREAD

**Discussion by ordo-n** [jcubic/jquery.terminal#959] [Q&A] [answered]
I'm hoping somebody can support me in determining what I'm doing wrong here, the text color doesnt invert the text color becomes the background color when highlighting text (selecting it)
![image](https://github.com/user-attachments/assets/9046a306-446f-4688-b0df-b4897c2574e0)

**Comment by ordo-n** [comment]
![image](https://github.com/user-attachments/assets/805532a9-1d2d-4597-a003-372985872a52)
![image](https://github.com/user-attachments/assets/d494406e-b814-4f13-93d6-f740dd0fd988)

**Comment by jcubic** [comment]
Do you have a demo Online? Something I can test?

**Reply by ordo-n** [reply]
the problem is --background:transparent;

<div id="terminal" style="width:100%;--background:transparent;"></div>

Is this intended functionality?

**Reply by ordo-n** [reply]
`<div id="terminal" style="width:100%;--background:transparent;"></div>`

**Reply by ordo-n** [reply, marked as answer]
The solution appears to be to use background:transparent instead of --background:transparent
